src/mixins/UserHasPermission.js

```
@@ -20,9 +20,12 @@ export default {
 	},
 	methods: {
 		$_userHasPermission(permission) {
-			return typeof permission === "string"
-				? !permission || this.userPermissions.includes(permission.toLowerCase())
-				: !permission() || permission(this.userPermissions);
+			if (permission)
```

I think this should throw an error if the permission is missing. The point of this method is to check if someone has a permission, if we do not provide a permission to this method, the complete call is pointless.

---

sure thing